Sources/PostgresNIO/New/PSQLError.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 06.01.21.
+//
+
+import struct Foundation.Data
```

Comment:
Ditch Xcode template headers.
```suggestion
import struct Foundation.Data
```